src/game.js

```diff
@@ -115,14 +120,18 @@ async function processEnteredCity(botContext) {
   if (selectedCity === null || selectedCity === undefined) {
     result.messages.push(
       botContext.translate("LOOSE_BOT") +
-        [...botContext.sessions[botContext.chatID].spentCities].join(", ")
+        [...botContext.sessions[botContext.chatID].spentCities].join(", ") +
```

Comment:
There is an [interpolation](https://www.i18next.com/translation-function/interpolation) feature in i18n that works much better for strings with value. Use it